travis/release.sh

```diff
@@ -5,6 +5,7 @@ if [ $TRAVIS_BRANCH == "cesium.com" ]; then
 else
   npm --silent run make-zip
   npm pack &> /dev/null
+  npm pack --workspaces &> /dev/null
```

Comment:
This was added to ensure the parent `cesium` package installs the working version of each workspace package, not the official published version.